fili-core/src/test/groovy/com/yahoo/bard/webservice/data/DruidResponseParserSpec.groovy

```diff
@@ -43,13 +36,17 @@ class DruidResponseParserSpec extends Specification {
 
     @Shared DimensionDictionary dimensionDictionary
```

Comment:
/NotABlockerBecauseYouDidn'tChangeIt

It would be _great_ to make this _not_ be shared so that we can't leak state across tests. (or at least not as easily)